uefi-mkconfig

```diff
@@ -27,8 +27,14 @@ add_uefi_entry () (
 
 	# Create path to initramfs
 	local initramfs_image
-	initramfs_image="${efi_file_path/${efi_file_path##*/}}initramfs-$kernel_version.img"
-	## Add .old suffix to initramfs in case we are handling kernel with -old suffix
+	if [[ -z $kernel_version ]]; then
+		# Use versionless initramfs image if the kernel efi file is versionless
+		initramfs_image="${efi_file_path/${efi_file_path##*/}}initramfs.img"
+	else
+		initramfs_image="${efi_file_path/${efi_file_path##*/}}initramfs-$kernel_version.img"
+	fi
```

Comment:
How about:
```
initramfs_image="${efi_file_path/${efi_file_path##*/}}initramfs${kernel_version:+-$kernel_version}.img
```

It's a bit shorter but should achieve the same effect.

Reply:
I would rather use the one in the commit. I find it more readable and extensible if need be.